dist/mapbox-gl.css

```diff
@@ -171,55 +236,61 @@ a.mapboxgl-ctrl-logo {
 
 .mapboxgl-ctrl.mapboxgl-ctrl-attrib {
     padding: 0 5px;
-    background-color: rgba(255, 255, 255, .5);
+    background-color: rgba(255, 255, 255, 0.5);
     margin: 0;
 }
+
 .mapboxgl-ctrl-attrib.mapboxgl-compact {
     padding-top: 2px;
     padding-bottom: 2px;
-    margin: 0 10px 10px 10px;
+    margin: 0 10px 10px;
     position: relative;
     padding-right: 24px;
     background-color: #fff;
     border-radius: 3px 12px 12px 3px;
     visibility: hidden;
 }
+
 .mapboxgl-ctrl-attrib.mapboxgl-compact:hover {
     visibility: visible;
 }
-.mapboxgl-ctrl-attrib.mapboxgl-compact:after {
+
+.mapboxgl-ctrl-attrib.mapboxgl-compact::after {
     content: '';
     cursor: pointer;
     position: absolute;
     bottom: 0;
     right: 0;
     background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg%20viewBox%3D%270%200%2020%2020%27%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%3E%0D%0A%09%3Cpath%20fill%3D%27%23333333%27%20fill-rule%3D%27evenodd%27%20d%3D%27M4%2C10a6%2C6%200%201%2C0%2012%2C0a6%2C6%200%201%2C0%20-12%2C0%20M9%2C7a1%2C1%200%201%2C0%202%2C0a1%2C1%200%201%2C0%20-2%2C0%20M9%2C10a1%2C1%200%201%2C1%202%2C0l0%2C3a1%2C1%200%201%2C1%20-2%2C0%27%20%2F%3E%0D%0A%3C%2Fsvg%3E");
-    background-color: rgba(255, 255, 255, .5);
+    background-color: rgba(255, 255, 255, 0.5);
     width: 24px;
     height: 24px;
     box-sizing: border-box;
     visibility: visible;
     border-radius: 12px;
 }
+
 .mapboxgl-ctrl-attrib a {
-    color: rgba(0,0,0,0.75);
+    color: rgba(0, 0, 0, 0.75);
     text-decoration: none;
 }
+
 .mapboxgl-ctrl-attrib a:hover {
     color: inherit;
     text-decoration: underline;
 }
-/* stylelint-disable */
```

Comment:
we still need to disable stylelint here and revert the `mapboxgl-improve-map` –> `mapbox-improve-map` per my previous comment